binder/src/main/java/io/grpc/binder/internal/PendingAuthListener.java

```diff
@@ -0,0 +1,126 @@
+package io.grpc.binder.internal;
+
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCall.Listener;
```

Comment:
I find it confusing to refer to the same class as both `Listener` and `ServerCall.Listener` in the same file. Consider standardizing on ServerCall.Listener since bare Listener feels quite ambiguous in grpc-java where there are so many types of listener.

Reply:
Updated.